.eslintrc.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
```

Comment:
Do we need plugin section?
Do we need env section with node:true?

Reply:
env added.
no plugins needed at this point. we might add them, once we'll have our own rules